docs/contributing/build-vyos.rst

```diff
@@ -23,6 +23,50 @@ also set up your own build machine and run a :ref:`build_native`.
    The source code remains public and an ISO can be built using the process
    outlined in this chapter.
 
+.. _build_native:
+
+Native Build
+============
+
+To build VyOS natively you require a properly configured build host with the
+following Debian versions installed:
+
+- Debian Jessie for VyOS 1.2 (crux)
+- Debian Buster for VyOS 1.3 (equuleus) 
+- Debian Bullseye for VyOS 1.4 (sagitta) 
+- Debian unknown for VyOS 1.5 (circinus, current) - aka the rolling release
```

Comment:
It's also Bookworm for current. I think a better wording might be "Debian Bookworm for the upcoming 1.5/Circinus (subject to change)".

Reply:
I am not very familiar with GitHub yet, I will make changes later. Thank you!